src/picosvg/svg_meta.py

```diff
@@ -192,3 +192,34 @@ def __new__(cls, attr_name):
 
     def __call__(self, data_obj):
         return getattr(data_obj, self.attr_name)
+
+
+_ATTRIB_DEFAULTS = {
```

Comment:
I believe we already listing the attributes' default values in the svg_types.py dataclasses; ideally we'd find a way to remove duplication and reuse

Reply:
oh sorry I see now you already are reusing the `attrib_default` in svg_types dataclasses, never mind my earlier comment

Reply:
I moved them.